CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -1,7 +1,30 @@
-# Release notes for 1.8.0
+# Release notes for 1.8.1
 
 [Documentation](https://kubernetes-csi.github.io)
 
+## Changes by Kind
+
+### Uncategorized
+
+- CVE fixes: CVE-2023-44487 (#343, @dannawang0221)
```

Comment:
Can you run release-notes with the "--markdown-links" flag? This way it will provide a markdown link for the PR and github id.